doc/api/dns.md

```diff
@@ -353,15 +353,15 @@ be an object with the following properties:
 * `minttl`
 
 ```js
-{
+({
```

Comment:
ESLint parses this as a code block with statements and breaks with parsing errors. With `()` it is parsed as an expression with an object literal.